### PR TITLE
added chrf signature; removed locale dependency

### PR DESCRIPTION
**Pull request by mjpost**
Update to 1.2.3:

command line:
- added chrF signature
- remove dependency on locale
- allow printing of multiple metrics

#### Pull Request Checklist ##
- [x] Changes are complete (if posting work-in-progress code, prefix your pull request title with '[WIP]'
until you can check this box.
- [x] Unit tests pass (`pytest`)
- [x] Were system tests modified? If so did you run these at least 5 times to account for the variation across runs?
- [x] System tests pass (`pytest test/system`)
- [x] Passed code style checking (`./pre-commit.sh` or manual run of pylint & mypy)
- [x] You have considered writing a test
- [x] Updated major/minor version in `sockeye/__init__.py`. Major version bump if this is a backwards incompatible change.
- [x] Updated CHANGELOG.md

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: contrib/sacrebleu/sacrebleu.py [hunk 1]
```diff
@@ -1228,12 +1271,14 @@ def main():
...
                             help='optional list of references (for backwards-compatibility with older scripts)')
-    arg_parser.add_argument('--metrics', '-m', choices=['bleu', 'chrf'], nargs='+', default='bleu',
+    arg_parser.add_argument('--metrics', '-m', choices=['bleu', 'chrf'], nargs='+', default=['bleu'],
                             help='metrics to compute (default: bleu)')
     arg_parser.add_argument('--chrf-order', type=int, default=CHRF_ORDER,
                             help='chrf character order (default: %(default)s)')
-    arg_parser.add_argument('--chrf-beta', type=float, default=CHRF_BETA,
+    arg_parser.add_argument('--chrf-beta', type=int, default=CHRF_BETA,
                             help='chrf BETA parameter (default: %(default)s)')
+    arg_parser.add_argument('--chrf-whitespace', action='store_true', default=not CHRF_REMOVE_WS,
```
Comment (davvil) [inline contrib/sacrebleu/sacrebleu.py]: Using 'store_true' together with a default dependent on a constant declared elsewhere is at least confusing and we could even end up with the default begin True.

Comment (mjpost) [reply contrib/sacrebleu/sacrebleu.py]: Are you suggesting we invert this? I want to default to excluding whitespace, but we could change the variable to `CHRF_ADD_WS` and invert the logic, if you think that's better and @fhieber signs off.

Comment (davvil) [reply contrib/sacrebleu/sacrebleu.py]: I would remove the "default=..." and the CHRF_REMOVE_WS constant altogether.

**Comment by mjpost** [comment]
Sorry, forgot to push them.

**Pull request merged**